src/constructs/cloudwatch/alarm.ts

```diff
@@ -0,0 +1,18 @@
+import type { AlarmProps } from "@aws-cdk/aws-cloudwatch";
+import { Alarm } from "@aws-cdk/aws-cloudwatch";
+import { SnsAction } from "@aws-cdk/aws-cloudwatch-actions";
+import type { ITopic } from "@aws-cdk/aws-sns";
+import { Topic } from "@aws-cdk/aws-sns";
+import type { GuStack } from "../core";
+
+export interface GuAlarmProps extends AlarmProps {
+  snsTopicArn: string;
```

Comment:
Most teams I've worked on have 1 or 2 topics configured for all alerts and then pass the `arn` into various templates as parameters.

We could also consider taking the topic name (which is not private) instead of the full `arn` (this contains an AWS account number, which should be kept private), to simplify things slightly?

Reply:
Yeah, it looks like that's what the [GuLogShippingPolicy](https://github.com/guardian/cdk/blob/964b73aa6a8f928a4d83de8427ae42fa7b825bd5/src/constructs/iam/policies/log-shipping.ts#L21) does. Then it gets the region and account from `scope`

Reply:
Is it worth having a single topic, set in param store, for use by multiple stacks? Or is the separation of concerns important?

Reply:
Good question. My initial thought is that configuring a single topic in parameter store would not work so well in this case because multiple teams sometimes share the same AWS account, but not the same topic for alerts (or the same mailing list). Similarly teams might want to use different topics for alerts of a different priority etc. 

I think it'd be worth thinking about how we might standardise this further in the future, but I'm inclined to leave it as is for now.